src/shared/styles/_ds-token.scss

```
@@ -0,0 +1,91 @@
+// =============================================================================
+// DS 3.1 토큰 진입점 (로컬 shim)
+// -----------------------------------------------------------------------------
+// 문제: @bigtablet/design-system 의 scss/token 은 colors/elevation 서브모듈을
+//   @forward 하는데, 두 모듈은 top-level 에 `:root { --bt-* }` / `[data-theme="dark"]`
+//   CSS 를 emit 한다. CSS Modules(*.module.scss)는 모듈마다 별도 컴파일되어 매번
+//   재emit → (1) 48× 중복, (2) `[data-theme="dark"]` 가 "not pure" 빌드 에러.
+//   DS 3.1.1 에는 emit 을 끄는 !default 플래그가 없고, exports 가 서브패스를
+//   막아 colors 만 선택 제외할 수도 없다.
+//
```

Stylelint (17.12.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)